src/state/slices/assetsSlice/selectors.ts

```diff
@@ -16,7 +17,10 @@ export const selectAssetNameById = createSelector(selectAssetByCAIP19, asset =>
   asset ? asset.name : undefined,
 )
 
-export const selectAssets = (state: ReduxState) => state.assets.byId
+export const selectAssets = createDeepEqualOutputSelector(
+  (state: ReduxState) => state.assets.byId,
+  byId => byId,
```

Comment:
`lodash.identity` would work also

Reply:
identity loses type information

Reply:
ah, weird, it doesn't. i tried this before and had issues. will change.

Reply:
actually, it returns `typeof state.assets.byId | undefined` which is incorrect. i agree - i'd like to use it too.